lib/colorls/core.rb

```diff
@@ -190,7 +190,7 @@ def mode_info(stat)
       format_mode(m >> 6, stat.setuid?, 's') +
         format_mode(m >> 3, stat.setgid?, 's') +
         format_mode(m, stat.sticky?, 't')
-    end
+    end    
```

Comment:
Rubocop does not like this change.

Reply:
Yes, you yourself are a robot! )

Reply:
No I am not. The Travis CI job failed because you violated [rubocop](https://www.rubocop.org/en/stable/)'s rules. When the CI fails, the PR is not mergable.